Cabal/tests/C2Hs/src/Codec/Archive/Types/Foreign.chs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric  #-}
```

Comment:
Is this file actually compiled, or is it only an input for tests?

Can we place it's somewhere where it doesn't look like real file.

I value that we have "real life" tests inputs, but we also should have something more minimal.

Also negative examples.